### PR TITLE
Optimize rendering

### PR DESCRIPTION
**Pull request by SoniEx2**

### CODE REVIEW

File: src/main/kotlin/gay/sylv/wij/impl/block/entity/render/WorldJarBlockEntityRenderer.kt [hunk 1]
```diff
@@ -126,23 +126,23 @@ class WorldJarBlockEntityRenderer(private val ctx: BlockEntityRendererFactory.Co
 			
 			// render each render layer
 			BLOCK_LAYERS.forEach { renderLayer ->
+				// set up shader
+				renderLayer.startDrawing()
+				val shader = RenderSystem.getShader()!!
```
Comment (sylv256) [inline src/main/kotlin/gay/sylv/wij/impl/block/entity/render/WorldJarBlockEntityRenderer.kt:131]: ah i completely forgot the shader

**Review by sylv256 — APPROVED** [review]
LGTM

**Comment by SoniEx2** [comment]
nvm looks like this has already been resolved XD

**Comment by sylv256** [comment]
i'll move the "getShader" thing since i forgot that

**Pull request closed without merging**